docs/book/v3/cookbook/using-a-base-path.md

```diff
@@ -26,9 +28,38 @@ RewriteEngine On
 RewriteRule (.*) ./public/$1
 ```
 
+### Using Nginx
+
+If you are using Nginx, add a named location block to rewrite to the Mezzio web root directory within the server context.
```

Comment:
A link to the Nginx documentation should be added where this part is described. Thanks!

Reply:
Added a link to the official docs. Hope that is what you are looking for.